dmutils/formats.py

```diff
@@ -65,11 +66,26 @@ def datetimeformat(value, default_value=None, localize=True):
       * dates that framework communication files were published
       * dates of revisions made to services (shown to admin users)
     """
-    formatted_date = _format_date(value, default_value, DISPLAY_DATETIME_FORMAT, localize=localize)
-    if formatted_date:
-        # en_GB locale uses uppercase AM/PM which contravenes our style guide
-        return formatted_date.replace('AM', 'am').replace('PM', 'pm').replace(" 0", " ")
-    return formatted_date
+    return _format_date(value, default_value, DISPLAY_DATETIME_FORMAT, localize=localize)
+
+
+def utctoshorttimelongdateformat(value, default_value=None, localize=True):
```

Comment:
It feels a bit weird that all the functions in here don't use snake case for their name like we do everywhere else.

Reply:
I think it's an old convention to distinguish it as a template filter - none of the Jinja built-in filters use snake case, for example. 

The old Django docs have disappeared off the internet now, but I think once upon a time Django custom template filters had naming restrictions because of how they were registered/scoped within the app. But I'm pretty sure there's no practical restriction on doing it there now. Or here!

Reply:
ah, interesting. Thanks Kat!